specification/schemas/HookLog.json

```diff
@@ -0,0 +1,121 @@
+{
+  "type": "object",
+  "required": [
+    "attributes",
+    "relationships"
+  ],
+  "properties": {
+    "attributes": {
+      "type": "object",
+      "required": [
+        "resource_diff",
+        "errors"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "resource_diff": {
+          "type": "object",
+          "example": {
+            "relationships" : {
+              "service": {
+                "data": {
+                  "id" : "7b808eee-bf1c-40cd-98f2-3c335a06417e"
```

Comment:
Maybe it's a good idea to also put the `type` property with value `services` in here, because that's what the hooks usually also set. There's no use in setting just an `id` or just a `type`.